specutils/analysis/flux.py

```diff
@@ -0,0 +1,87 @@
+from __future__ import division
+
+import numpy as np
+from astropy.units.quantity import Quantity
+from .utils import computation_wrapper
+
+
+__all__ = ['line_flux', 'equivalent_width']
+
+
+def line_flux(spectrum, region=None):
+    """
+    Computes the line flux over a spectrum. Applies a region if given.
```

Comment:
Maybe change to "Computes the flux in a spectrum or area of a spectrum. 
 Applies to the whole spectrum by default, but can be limited to a specific feature (like a spectral line) if a `region` is given."